packages/block-library/src/group/variations.js

```diff
@@ -24,7 +24,7 @@ const variations = [
 		title: _x( 'Row', 'single horizontal line' ),
 		description: __( 'Arrange blocks horizontally.' ),
 		attributes: { layout: { type: 'flex', flexWrap: 'nowrap' } },
-		scope: [ 'block', 'inserter', 'transform' ],
```

Comment:
Ah, I think I see now. Is this removal to hide the transforms at the top of the sidebar? It also seems to hide the multiple block selection grouping buttons, as well as preventing blocks from being transformed to the different variations.

Reply:
Yeah, I wanted to disable the top of sidebar transforms specifically. Is there any way to disable them without those side-effects?

Reply:
I don't think there is at the moment, but it looks like the logic is currently to grab the variations based on `transform` here: https://github.com/WordPress/gutenberg/blob/trunk/packages/block-editor/src/components/block-variation-transforms/index.js#L112 — so if we want to conditionally hide it for a particular block, we might need an update somewhere around that file? I'm not sure how it'd ideally be flagged in the variations, but hope that helps!